docs/current_docs/integrations/snippets/tekton-dagger-task.yaml

```diff
@@ -46,7 +43,7 @@ spec:
     script: |
       #!/usr/bin/env sh
       apk add curl
-      curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=$HOME/.local/bin sh
+      curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=/usr/local/bin sh
```

Comment:
Just curious about why this change? We use $HOME in the official CLI download instructions

Reply:
$HOME is appropriate for workstation installs but not CI. `$HOME/.local/bin` isn't going to be in the path in CI